spt/spt-serverplugin.cpp

```diff
@@ -262,6 +271,13 @@ bool CSourcePauseTool::Load(CreateInterfaceFn interfaceFactory, CreateInterfaceF
 
 void CSourcePauseTool::Unload(void)
 {
+	if (skipUnload)
+	{
+		// Preventing double load of plugin, do nothing on unload
+		skipUnload = false; // Enable unloading again
+		return;
+	}
+
 #if !defined(OE)
```

Comment:
Should this also set pluginLoaded = false?

Reply:
No, when this occurs the plugin is still loaded and is running on the same instance of that DLL if that makes sense.

Reply:
I mean when you unload the last copy.

Reply:
It still unloads and loads fine, in a real unload the library gets freed completely it seems.
![image](https://user-images.githubusercontent.com/10210201/146980144-3779b851-7bce-418c-b4ae-cc84bcdd526a.png)

Reply:
ah, cool